packages/mint-components/src/components/tax-and-cash/sqm-tax-form/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+
+Feature: Tax Document Submitted View
+        View that informs the user of their Tax Document submission status.
+
+	Background: A user has submitted their Tax Document
+		Given a user is on the Tax Document Submitted view
+
+	@motivating
+	Scenario: View Tax Document Submitted
+		Given the user has submitted a tax document
+		When they view Document Submitted
+		Then they should see the status of their Tax Document and Banking Information
+
+	@minutia
+	@ui
+	Scenario Outline: The Tax Form header is controlled via <documentType> prop
+	Given the <documentType>
```

Comment:
When writing a scenario outline think about how the step sounds with the variable filled in (e.g. "Given the W9"). Tricky to find the right language sometimes but something like "Given a W9/W8-BEN form" would be a bit easier to read.